packages/mds-provider-service/repository/fallback_providers.ts

```diff
@@ -0,0 +1,346 @@
+import type { MDSProviderDomainModel } from '../@types'
```

Comment:
Could we async load the fallback from Github instead of hard coding?

Reply:
I think at the very least we'd need to have fallback + extension, because there are quite a few cases where we have provider_ids registered before they're OMF ratified (e.g. all Taxi Co's). I suspect that it'll take some time for us to start really leaning on this repository in production, so we should ensure that all of our currently supported providers will still show up about the same as they do today.

Reply:
I think adding this with a migration would be fine, though it's worth noting that in some tenants we will want there to be a partial list in the DB in the near-future. Ideally, tenants only have the provider entries they need in them. So this will require us to do some manual cleanup down the road to delete the unused entries in each environment.

I think that what'd make sense is to use the fallback list for now, get the config files configured for each tenant, add the repo seeding functionality to this service, and then delete the fallback once that's all ready.

Reply:
Adding a note here just so it's recorded: @drtyh2o and @avatarneil discussed it outside of this PR and we're going to go with seeding the first providers via ConfigMaps and avoid using migrations and go with a hardcoded list of fallback providers.